tools/webidl_binder.py

```diff
@@ -546,7 +547,7 @@ def is_ptr_arg(i):
   c_names = {}
 
   def make_call_args(i):
-    if pre_arg:
+    if has_pre_arg:
```

Comment:
I don't see how this is any different to `bool(pre_arg)` which itself is identical to `pre_arg` in an `if` statement, no?

Reply:
There's an assignment above that does `call_args = pre_arg`, then we append to `call_args` which modifies pre_arg as well.

Reply:
Alternatively, we could instead clone/copy pre_arg to call_args so modifying one does not modify the other.

Reply:
Ah, in that case I think its better to do `call_args = list(pre_arg)` or `call_args = copy(pre_arg)` above.

Reply:
oops `call_args = pre_arg.copy()`